README.md

```diff
@@ -181,6 +181,15 @@ versioning](https://semver.org/). In other words:
 - Changes which break backwards compatibility: Major release, increment the
   first number, e.g. 2.0.0
 
+Bumping versions
+--------
+1. Starting in the  `development`  branch, update the version number in the `package.json`  on line 3
+2.  `npm install`
+3. Then run  `bundle install`  to update the  `Gemfile.lock`
+4. Push up changes and merge into  `development` and a TeamCity build will fire.
+5. Once the build passes, merge the changes into `master`.
+6. After merging into `master`  goto TeamCity and find “Styles Deploy” under “Release” (You should see that there are pending changes). Then you want to click the “Run” button. This build will update the release number in GitHub and NPM and it will update files in an S3 bucket
```

Comment:
@silasbaur Sorry, one more comment that did not save. I would move "Then you want to click the..." to step number 7. I personally feel it would be easier to read